src/test/java/org/openrewrite/kotlin/tree/LambdaTest.java

```diff
@@ -67,4 +69,34 @@ fun inputValues ( ) : List < Pair < String , Any ? > > {
           )
         );
     }
+
+    @Issue("https://github.com/openrewrite/rewrite-kotlin/issues/60")
+    @Test
+    @Disabled
```

Comment:
We could adopt `@ExpectedToFail` here from junit-pioneer; that way the test will always run, and fail only when the issue is fixed, to signify that we can update and close the associated issue. Optional, but potentially nice to have.

Reply:
Sounds like a good idea! Shall I do the same for test cases added in https://github.com/openrewrite/rewrite-kotlin/pull/55 and https://github.com/openrewrite/rewrite-kotlin/pull/57?

They're a bit unrelated to this PR.. I can also split it if preferred ☺️

Reply:
Fine to do that here & Thanks for the quick response!